core/tools/dependencies/cleanup.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Immediately exit on errors
+set -e
+
+apt -y remove ${BUILD_PACKAGES[*]}
```

Comment:
Variables are not accessible through shellscripts.

Reply:
Didn't get it.

Reply:
BUILD_PACKAGES is not defined in this script the variable need to be defined on env to be accessible.